gatsby/src/components/Article.tsx

```diff
@@ -0,0 +1,60 @@
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import styles from './Article.module.scss';
+import H1 from './mdx/H1';
+import TableOfContents, { ITableOfContents } from './TableOfContents';
+
+interface IArticle {
+  children: React.ReactNode;
+  tableOfContents?: ITableOfContents;
+  title: string;
+}
+
+const Article: React.FC<IArticle & React.HTMLProps<HTMLElement>> = (props) => {
+  const { children, className, tableOfContents, title, ...rest } = props;
+  const currentItem = '#redshift-is-hard';
```

Comment:
This probably shouldn't be hardcoded

Reply:
The table of contents functionality isn't fully implemented, and I was anticipating this functionality to be expanded upon later.